<macports>Patch from debian: http://patch-tracker.debian.org/package/eperl/2.2.14-16</macports>
Fix some spelling errors found by lintian as well as a trailing whitespace.

Index: eperl/mod/Apache/ePerl.pm
===================================================================
--- eperl.orig/mod/Apache/ePerl.pm	2010-06-05 15:59:27.000000000 +0200
+++ eperl/mod/Apache/ePerl.pm	2010-06-05 15:59:58.000000000 +0200
@@ -432,14 +432,14 @@
 
 =head1 DESCRIPTION
 
-This packages provides a handler function for Apache/mod_perl which can be
+These packages provides a handler function for Apache/mod_perl which can be
 used to emulate the stand-alone Server-Side-Scripting-Language I<ePerl> (see
 eperl(3) for more details) in a very fast way. This is not a real 100%
 replacement for F<nph-eperl> because of reduced functionality under some
 special cases, principal runtime restrictions and speedup decisions. For
 instance this variant does not (and cannot) provide the SetUID feature of
 ePerl nor does it check for allowed filename extensions (speedup!), etc.
-Instead it uses further features like object caching which ePerl does not use. 
+Instead it uses further features like object caching which ePerl does not use.
 
 But the accepted bristled source file format is exactly the same as with the
 regular ePerl facility, because Apache::ePerl uses the Parse::ePerl package
@@ -527,7 +527,7 @@
 in December 1996 as a plain Perl module after he has seen the original ePerl
 from Ralf S. Engelschall. It implemented the ePerl idea, but was not
 compatible to the original ePerl. In May 1997 Hanno Mueller
-E<lt>hmueller@kabel.deE<gt> has taken over the maintainance from Mark I. and
+E<lt>hmueller@kabel.deE<gt> has taken over the maintenance from Mark I. and
 enhanced Apache::ePerl by adding caching for P-Code, adding the missing
 C<chdir> stuff, etc. 
 
